charts/tempo-distributed/values.yaml

```diff
@@ -577,14 +592,57 @@ traces:
   # -- Enable Tempo to ingest traces from Kafka. Reference: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kafkareceiver
   kafka: {}
 
-
 # -- Config file contents for Tempo distributed. Passed through the `tpl` function to allow templating
 
 # @default -- See values.yaml
 config: |
   multitenancy_enabled: {{ .Values.multitenancyEnabled }}
   search_enabled: {{ .Values.search.enabled }}
   metrics_generator_enabled: {{ .Values.metricsGenerator.enabled }}
+
```

Comment:
Optional: this is subtle stuff, but in mimir-distributed I reordered the yaml in `config` template alphabetically because the processing we do will output the result in an ordered way.

Reply:
A very interesting.